mscore/keyb.cpp

```diff
@@ -213,10 +213,10 @@ void ScoreView::editKey(QKeyEvent* ev)
             }
 
 
-#ifdef Q_OS_WIN // Japenese IME on Windows needs to know when Contrl/Alt/Shift/CapsLock is pressed while in predit
+#ifdef Q_OS_WIN // ignore Contrl/Alt/Shift/CapsLock while in predit
```

Comment:
I don't know if the string kill glitch is present on Mac and Linux.  If so, then it might also make sense to remove this ifdef.